src/js/brushing/mg_line_brushing.js

```diff
@@ -0,0 +1,346 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['d3', 'jquery', 'MG'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('d3'), require('jquery'), require('MG'));
+  } else {
+    root.Mg_line_brushing = factory(root.d3, root.jQuery, root.MG);
+  }
+}(this, function(d3, $, MG) {
```

Comment:
I don't believe this type of commonjs wrapper is needed for anything that's being compiled into metricsgraphics itself. I would use the example of other files.